app/gui/src/project-view/components/shared/AgGridTableView.vue

```diff
@@ -1,3 +1,42 @@
+<script lang="ts">
+import { gridBindings } from '@/bindings'
+import type { MenuItemDef } from 'ag-grid-enterprise'
+/**
+ * A more specialized version of AGGrid's `MenuItemDef` to simplify testing (the tests need to provide
+ * only values actually used by the composable)
+ */
+export interface MenuItem<TData> extends MenuItemDef<TData> {
+  action: (params: {
+    node: { data: TData | undefined } | null
+    api: { copyToClipboard: () => void; cutToClipboard: () => void; pasteFromClipboard: () => void }
+  }) => void
+}
+
+const AGGRID_DEFAULT_COPY_ICON =
+  '<span class="ag-icon ag-icon-copy" unselectable="on" role="presentation"></span>'
+
+export const commonContextMenuActions = {
+  cut: {
+    name: 'Cut',
+    shortcut: gridBindings.bindings['cutCells'].humanReadable,
+    action: ({ api }) => api.cutToClipboard(),
+    icon: AGGRID_DEFAULT_COPY_ICON,
+  },
+  copy: {
+    name: 'Copy',
+    shortcut: gridBindings.bindings['copyCells'].humanReadable,
+    action: ({ api }) => api.copyToClipboard(),
+    icon: AGGRID_DEFAULT_COPY_ICON,
+  },
+  paste: {
+    name: 'Paste',
+    shortcut: gridBindings.bindings['pasteCells'].humanReadable,
+    action: ({ api }) => api.pasteFromClipboard(),
+    icon: AGGRID_DEFAULT_COPY_ICON,
```

Comment:
Should these three `icon` fields all have the same value?

Reply:
They are the same in the "original" menu items of AgGrid